src/applications/gi/gi-entry.jsx

```diff
@@ -5,7 +5,7 @@ import startApp from '../../platform/startup';
 
 import routes from './routes';
 import reducer from './reducers';
-import manifest from './manifest';
+import manifest from './manifest.json';
```

Comment:
is json preferred?

Reply:
I think so. We don't need the functionality in the js version anymore, so I think json is simpler.

I changed this because I was trying different things. Left it because I think it's worth converting these configs back to json.